js_modules/dagster-ui/packages/ui-core/src/ui/Filters/useDefinitionTagFilter.tsx

```diff
@@ -11,7 +11,7 @@ import {buildTagString} from '../tagAsString';
 
 const emptyArray: any[] = [];
 
-type Tag = Omit<DefinitionTag, '__typename'>;
+export type Tag = Omit<DefinitionTag, '__typename'>;
```

Comment:
Seems like a bit of an overloaded name to export. Maybe it can be something a little more specific, `DefinitionTagType` or something?

Reply:
I think it's not a big deal since the file path acts as a namespace